perf/lua/CMakeLists.txt

```diff
@@ -31,22 +32,37 @@ function(create_perf_lua_test)
   # or not, and the second target (BENCH_RESULT_TARGET) is run
   # only if the corresponding file is omitted. The COMMAND_LIST
   # variable contains the same command for these targets.
+  set(TEST_DEPS tarantool ${PERF_DEPENDS} ${TEST_PATH})
   set(COMMAND_LIST
         COMMENT Running ${BENCH_TARGET}
         COMMAND ${CMAKE_COMMAND} -E env
           LUA_PATH="${LUA_PATH}"
           ${BENCH_CMD_SEPARATE} ${TARANTOOL_BIN} ${TEST_PATH}
             --output="${BENCH_RESULT}"
             --output_format=json
-        DEPENDS tarantool ${PERF_DEPENDS} ${TEST_PATH}
+        DEPENDS ${TEST_DEPS}
         WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
   )
+
   add_custom_command(OUTPUT ${BENCH_RESULT} ${COMMAND_LIST})
   add_custom_target(${BENCH_RESULT_TARGET} DEPENDS ${BENCH_RESULT})
   add_custom_target(${BENCH_TARGET} ${COMMAND_LIST})
 
+  # XXX: The call produces both test and target
+  # <${PERF_NAME}-deps> as a side effect.
+  _add_test_suite_target(${PERF_NAME}
+    LABELS "${TEST_SUITE_NAME};performance;"
+    DEPENDS ${TEST_DEPS})
   set(RUN_PERF_LUA_TESTS_LIST ${RUN_PERF_LUA_TESTS_LIST} ${BENCH_TARGET} PARENT_SCOPE)
   set(BENCH_RESULTS ${BENCH_RESULT_TARGET} ${BENCH_RESULTS}  PARENT_SCOPE)
+  set(test_title "perf/lua/${PERF_NAME}.lua")
+  add_test(NAME ${test_title}
+           COMMAND ${TARANTOOL_BIN} ${CMAKE_CURRENT_SOURCE_DIR}/${PERF_NAME}.lua
```

Comment:
Does it avoid arguments for tests intentionally?
The result is just running the perf test itself, but to see the result, it is necessary to add the `--verbose` option (the `perf/output` directory is empty too, obviously). 

```
1902: Test command: /home/burii/reviews/tarantool/ligurio/src/tarantool "/home/burii/reviews/tarantool/ligurio/perf/lua/uri_escape_unescape.lua"
1902: Working Directory: /home/burii/reviews/tarantool/ligurio/perf/lua
1902: Environment variables: 
1902:  LUA_PATH=/home/burii/reviews/tarantool/ligurio/perf/lua/?.lua;;
1902: Test timeout computed to be: 1500
1902: uri.escape 5830 rps
1902: uri.unescape 4492 rps
```

Reply:
Fixed:

```diff
diff --git a/perf/CMakeLists.txt b/perf/CMakeLists.txt
index 235a6990b8..e695dffb10 100644
--- a/perf/CMakeLists.txt
+++ b/perf/CMakeLists.txt
@@ -89,7 +89,10 @@ function(create_perf_test_target)
   set(BENCH_RESULTS ${BENCH_RESULT_TARGET} ${BENCH_RESULTS} PARENT_SCOPE)
   set(test_title "perf/${PERF_TARGET}.cc")
   add_test(NAME ${test_title}
-           COMMAND "$<TARGET_FILE:${PERF_TARGET}.perftest>")
+           COMMAND "$<TARGET_FILE:${PERF_TARGET}.perftest>"
+                   --benchmark_out_format=json
+                   --benchmark_out="${BENCH_RESULT}"
+  )
   set_tests_properties(${test_title} PROPERTIES
     LABELS "${TEST_SUITE_NAME};performance;"
     DEPENDS ${TEST_SUITE_NAME}-deps
diff --git a/perf/lua/CMakeLists.txt b/perf/lua/CMakeLists.txt
index f228e573d0..d4790f183f 100644
--- a/perf/lua/CMakeLists.txt
+++ b/perf/lua/CMakeLists.txt
@@ -57,7 +57,9 @@ function(create_perf_lua_test)
   set(BENCH_RESULTS ${BENCH_RESULT_TARGET} ${BENCH_RESULTS}  PARENT_SCOPE)
   set(test_title "perf/lua/${PERF_NAME}.lua")
   add_test(NAME ${test_title}
-           COMMAND ${TARANTOOL_BIN} ${CMAKE_CURRENT_SOURCE_DIR}/${PERF_NAME}.lua
+           COMMAND ${TARANTOOL_BIN}
+                   ${CMAKE_CURRENT_SOURCE_DIR}/${PERF_NAME}.lua
+                   --output=${BENCH_RESULT} --output_format=json
   )
   set_tests_properties(${test_title} PROPERTIES
     ENVIRONMENT "LUA_PATH=${LUA_PATH}"
```